manifests/state-ofed-driver/0050_ofed-driver-ds.yaml

```diff
@@ -146,6 +154,30 @@ spec:
             initialDelaySeconds: {{ .CrSpec.ReadinessProbe.InitialDelaySeconds }}
             failureThreshold: 1
             periodSeconds: {{ .CrSpec.ReadinessProbe.PeriodSeconds }}
+        {{- if .RuntimeSpec.UseDtk }}
```

Comment:
do we support requests/limits for this container ?

Reply:
Added requests/limits for this container